src/components/ValidatedInput/ValidFieldText.tsx

```diff
@@ -256,15 +258,19 @@ export class ValidFieldText extends React.Component<ValidFieldTextProps, State>
           )}
           {!!maxLength && <FieldCounter value={value} maxLength={maxLength} className="px-0" />}
         </div>
-        <div className={classNames("valid-field-text-input-container", { focusedFieldText: this.state.isFocus })}>
-          {popoverContent ? (
-            <InlinePopover isOpen={isFocus} placement={popoverPlacement} content={popoverContent} type={popoverType}>
-              {this.renderInput()}
-            </InlinePopover>
-          ) : (
-            this.renderInput()
-          )}
-          {!!rightIcon && <div className="rightIcon">{rightIcon}</div>}
+
+        <div className="field-container">
```

Comment:
have you made sure this doesn't break any validfieldtext style in AP?

Reply:
Not yet, let me check it in my local